### PR TITLE
upgraded requests

### PR DESCRIPTION
**Pull request by anahocevar**

### CODE REVIEW

**Review by ZachGlassman — APPROVED** [review]
I think this is failing because we didn't lock the version of detect secrets.  That said, this change should be fine.

**Comment by anahocevar** [comment]
I see. So are we good to merge as is?

**Comment by ZachGlassman** [comment]
yes.  If you want to you can try to lock the version of detect secrets to something older, maybe 0.9.1?

File: dev-requirements.txt [hunk 1]
```diff
@@ -1 +1 @@
-detect-secrets
+detect-secrets = 0.9.1
```
Comment (ZachGlassman) [inline dev-requirements.txt]: I think for pip its two equal signs

**Comment by ZachGlassman** [comment]
Merge Away!

**Pull request merged**